project/AkkaBuild.scala

```diff
@@ -98,13 +98,13 @@ object AkkaBuild extends Build {
     aggregate = aggregatedProjects
   ).settings(rootSettings: _*)
 
-  lazy val akkaScalaNightly = Project(
-    id = "akka-scala-nightly",
-    base = file("akka-scala-nightly"),
-    // remove dependencies that we have to build ourselves (Scala STM)
-    // samples don't work with dbuild right now
-    aggregate = aggregatedProjects diff List(agent, docs, samples)
-  ).disablePlugins(ValidatePullRequest, MimaPlugin)
+//  lazy val akkaScalaNightly = Project(
```

Comment:
Add a TODO or a description on why commented out (so we know when we look at this file and see this commented out)